backend/src/main/conf/local.json

```diff
@@ -1,11 +1,27 @@
 {
+  "backend.http.server": {
+    "port": 9300,
+    "idle.timeout.secs": 60
+  },
+  "leader.http.server": {
+    "port": 9400,
+    "idle.timeout.secs": 60
+  },
+  "http.client": {
+    "connect.timeout.ms": 5000,
+    "idle.timeout.secs": 10,
+    "max.attempts": 3,
+    "keepalive": true,
+    "compression": true
+  },
+  "load.report.interval.secs": 2,
   "vertxOptions": {
     "blockedThreadCheckInterval": 60000
```

Comment:
Configuration in this file is for localhost testing and does not reflect values going to be in production.